web-common/src/layout/index.ts

```diff
@@ -1 +1,12 @@
 export { default as RillTheme } from "./RillTheme.svelte";
+import { browser } from '$app/environment'
+import '../lib/i18n' // Import to initialize. Important :)
+import { locale, waitLocale } from 'svelte-i18n'
+import type { LayoutLoad } from './$types'
+
+export const load: LayoutLoad = async () => {
+	if (browser) {
+		locale.set(window.navigator.language)
+	}
+	await waitLocale()
+}
```

Comment:
1. Can you encapsulate all the i18n initialization logic (both the logic in this file and the logic in `lib/i18n/index.ts`) into one function `initializei18n()` and call the function from the top of `web-local/src/routes/+layout.svelte`?
2. No need for any checks for `if(browser)`. Because we're running SvelteKit in SPA-mode, there's no SSR and `browser` will always be `true`.

Reply:
ok, I started this way and then afterwards though I would only need translation in web-common.  I can make the change.

Reply:
The function can live in `web-common` – then we can reuse it easily in the other apps (`web-admin`, `web-auth`). Just moving its invocation to `+layout.svelte` will make it very explicit that it's one of the functions that's called on startup.